### ISSUE TITLE
Modal Focus not Handled

### ISSUE THREAD

**Issue by markaplet** [Office-of-Digital-Services/California-State-Web-Template#136]
I noticed today while using a screen reader (NVDA) that when a modal is activated that focus is not sent to the modal. Specifically I expected focus to go to the title. Pressing the down arrow with the modal open I was expecting NVDA to read the contents of the modal, however it read only the content from the page behind the modal. Pressing tab users are unable to refocus the modal in order to close it. 

In a different scenario, opening the modal then pressing tab allowed the close button to be focused. From that point, users can use the down arrow to read the content of the modal but this is unexpected / unintended behavior. 

I believe the focus is not going to the modal because it lacks  tabindex -1 attribute. 

See https://getbootstrap.com/docs/4.6/components/modal/ for modal documentation

**Comment by kkoryaka** [comment]
@markaplet, thank you for your feedbacks. and for alwAys bringing great points. Recently, we have switched our template to bootstrap 5 which means codebase for most of our components are now updated. Take a look again and see if your issue is relevant. Our engineers are also taking thorough look at it to see if there are any potential accessibility issues.

**Comment by yesenia-gtz** [comment]
@kkoryaka  OK to close this out?

**Comment by kkoryaka** [comment]
@yesenia-gtz I just closed it. Thanks
